snuba/admin/static/index.tsx

```diff
@@ -19,6 +20,19 @@ const bodyStyle = {
 };
 
 let client = Client();
+client.getSettings().then((settings) => {
+  if (settings.dsn != "") {
+    Sentry.init({
+      dsn: settings.dsn,
+      integrations: [new Sentry.BrowserTracing(), new Sentry.Replay()],
+      // Performance Monitoring
+      tracesSampleRate: 1.0, // Capture 100% of the transactions, reduce in production!
```

Comment:
How do we reduce this in production, shouldn't we just set the right value here?

Reply:
I wasn't planning to reduce it at all. This isn't going to generate a lot of traces. However I will change it to be configurable just in case.